Solutions/CrowdStrike Falcon Endpoint Protection/Data Connectors/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json

```diff
@@ -2,7 +2,7 @@
     "id": "CrowdStrikeFalconEndpointProtection",
     "title": "CrowdStrike Falcon Endpoint Protection",
     "publisher": "CrowdStrike",
-    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
+    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Microsoft Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
```

Comment:
Please check if we are missing to add any content as highlighted in `blue color brackets`
![image](https://user-images.githubusercontent.com/107389644/174785074-ce754492-4ed6-4ce4-bdca-87564c96a9f2.png)